test/source/tests/page-recipe/setup-page-recipe.ts

```diff
@@ -108,9 +108,9 @@ export class SetupPageRecipe extends PageRecipe {
   ) {
     if (!noPrvCreateOrgRule) {
       if (usedPgpBefore) {
-        await settingsPage.waitAndClick('@action-step0foundkey-choose-manual-enter', { retryErrs: true });
+        await settingsPage.waitAndClick('@action-step0foundkey-choose-manual-enter', { timeout: 30000, retryErrs: true });
```

Comment:
Unfortunately, yes. I'll explain the reason in another issue because we might want to fix it as the initial loading state during the setup process might take more than 20s in some cases.

Reply:
Yes, I meant, should there be `30` instead of `30000`?

Reply:
Right, fixed 👍 

The issue about the slowness of that page is reported here: https://github.com/FlowCrypt/flowcrypt-browser/issues/4102